backend/tests/integration/test_mcp_client_masking.py

```
@@ -0,0 +1,518 @@
+- MCPClient and DataMaskingService integration
+- Configuration flow from registry to masking
+- Real masking behavior in call_tool responses
+- Error handling in integrated scenarios
+"""
+
+import pytest
+from unittest.mock import Mock, AsyncMock, patch
+
+from tarsy.config.settings import Settings
+from tarsy.integrations.mcp.client import MCPClient
+from tarsy.models.mcp_config import MCPServerConfig
```

Ruff (0.12.2)

`tarsy.models.mcp_config.MCPServerConfig` imported but unused

Remove unused import: `tarsy.models.mcp_config.MCPServerConfig`

(F401)